tests/integration/utils.ts

```diff
@@ -41,7 +41,7 @@ export async function getStartedTestContainer(
     process.env.TESTCONTAINERS_NODE_IMG || 'kiltprotocol/standalone-node'
   console.log(`using testcontainer with image ${image}`)
   const strategies = [
-    ['--dev', '--ws-external', `--ws-port=${WS_PORT}`],
+    // ['--dev', '--ws-external', `--ws-port=${WS_PORT}`],
```

Comment:
Why this?

Reply:
Trying to figure out why the test container isn't loading. To active the CI and see what is happening and why the container isn't loading.